templates/configmaps_overrides.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.st2.override.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-st2-override-configs
+  annotations:
+    description: StackStorm override configs defined in helm values, shipped in (or copied to) '/opt/stackstorm/overrides'
+  labels:
+    app: st2
+    tier: backend
+    vendor: stackstorm
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+{{ toYaml .Values.st2.override.configs | indent 2 }}
+{{- end }}
```

Comment:
please keep the newline at the end of file

Reply:
New line added!